translator-deepl/src/main/java/org/xwiki/contrib/translator/deepl/DeeplTranslator.java

```diff
@@ -56,6 +91,10 @@ public String translate(String content, Locale from, Locale to, boolean html) th
         if (html) {
             options.setTagHandling("html");
         }
+        Optional<String> glossaryId = getGlossaryIdForLocales(from, to);
+        if (glossaryId.isPresent()){
```

Comment:
Missing space before the bracket -> was the XWiki Java code style correctly  applied?